src/templates/app.html

```diff
@@ -26,7 +26,7 @@
     <script>
       window.AFFINITY_DEFAULT_TEAM = "<%= VUE_APP_DEFAULT_TEAM %>";
     </script>
-    <div id="app"></div>
+    <div id="app" class="dark"></div>
```

Comment:
```suggestion
    <div id="app"></div>
```

I don't think we need this.

Reply:
Resolved